### PR TITLE
Updates the pull request template to remove boilerplate.

### PR DESCRIPTION
**Pull request by skellock**
## Please verify the following:
- [ ] Everything works on iOS/Android
- [ ] `ignite-base` **ava** tests pass
- [ ] `fireDrill.sh` passed

## Describe your PR
Now we won't see checklists appear in the issue list.  And people can use their own checklists for the purposes of good.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
+vvvvvv Please remove this section before submitting vvvvvv
+
 ## Please verify the following:
 - [ ] Everything works on iOS/Android
 - [ ] `ignite-base` **ava** tests pass
```
Comment (GantMan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Won't this be dead in 2.0?

Comment (skellock) [inline .github/PULL_REQUEST_TEMPLATE.md]: Yes.  We'll be using CI to enforce these rules automatically.  No need to put them in the PR template, imho.

Comment (GantMan) [inline .github/PULL_REQUEST_TEMPLATE.md]: I say we overhaul our templates.   They were gut reactions that need review.

**Comment by jamonholmgren** [comment]
Fixed in #849

**Pull request closed without merging**